mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+let productTax = (productPrice*20) / 100 + productPrice;
```

Comment:
Don't forget to indent. If you set up prettier and 'format on save' correctly, then this should happen automatically: https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week1/tree/master/exercises/A-setup-ide

Let me know if you need any help with that